sdk/include/psdk/ntddstor.h

```diff
@@ -169,8 +169,11 @@ DEFINE_DEVPROPKEY(DEVPKEY_Storage_Gpt_Name,           0x4d1ebee8, 0x803, 0x4774,
 #define IOCTL_STORAGE_PERSISTENT_RESERVE_IN \
   CTL_CODE(IOCTL_STORAGE_BASE, 0x0406, METHOD_BUFFERED, FILE_READ_ACCESS)
 
+#ifndef IOCTL_STORAGE_PERSISTENT_RESERVE_OUT_DEFINED
+#define IOCTL_STORAGE_PERSISTENT_RESERVE_OUT_DEFINED
```

Comment:
Question for these sort of new `ifndef xxx_DEFINED #define xxx_DEFINED ...` which now pollute our headers, instead of the `__WINESRC__` before:
Is it possible to avoid these altogether?
For example for this one, only this definition is guarded this way, but the other place where `IOCTL_STORAGE_PERSISTENT_RESERVE_OUT` is defined is in wine's `sdk/include/reactos/wine/winioctl.h` , which however, defines also all the other defines that you haven't guarded. So why wouldn't there be a problem for the other defines, except for this one precisely? etc.

Reply:
Deleted `#ifndef ..._DEFINED` etc. Build is broken now.